sample_data_config/elife-article-xml/elife-article-xml.config.yaml

```diff
@@ -8,7 +8,24 @@ elifeArticleXml:
           # parameters will be skipped if referenced file does not exist
           - parameterName: Authorization
             filePathEnvName: GITHUB_API_AUTHORIZATION_FILE_PATH
+      # XML elements that we are interested in (by replaceing '-' with '_')
+      searchedXmlElements: ['related_article', 'article_id', 'article_categories']
```

Comment:
Actually I just realise, `selected` might be a better terms. And maybe `Element` is slightly confusing because it happens after conversion which you documented. But maybe it would be better represented in the name? e.g. `selectedNormalizedXmlElements` or similar?